ingest/workflow/snakemake_rules/nextclade.smk

```diff
@@ -7,6 +7,33 @@ rule nextclade_dataset:
         nextclade dataset get --name MPXV --output-zip {output}
         """
 
+rule nextclade_dataset_hMPXV:
+    output:
+        temp("hmpxv.zip")
+    shell:
+        """
+        nextclade dataset get --name hMPXV --output-zip {output}
+        """
+
+rule align:
+    input:
+        sequences = "data/sequences.fasta",
+        dataset = "mpxv.zip"
```

Comment:
Is this supposed to be `hmpxv.zip`? Only asking because the hMPXV dataset was added above but doesn't look like it's used anywhere.

Reply:
thanks, Jover. align now uses hmpxv. shouldn't matter much in practice, but I think people will be more confidence when the alignment is based on an actual sequence, rather than a reconstructed ancestor.